client/lib/main.less

```diff
@@ -660,6 +660,36 @@ iframe.js {
       background: @brand-color;
     }
 
+    &.donations {
+      background: @brand-color;
+      animation: rotate-color 30s linear infinite;
```

Comment:
I find the color cycling a bit distracting, but my wife loves it. That's the extent of my user testing so far. Could you tweak the transition time from 30s to 180s?